crates/testlang/outputs/npm/package/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@slang-private/slang-testlang",
+  "version": "0.13.1",
+  "description": "A modular set of compiler APIs empowering the next generation of Testlang code analysis and developer tooling. Written in Rust and distributed in multiple languages.",
+  "homepage": "https://nomicfoundation.github.io/slang/",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NomicFoundation/slang.git"
+  },
+  "bugs": "https://github.com/NomicFoundation/slang/issues/",
+  "contributors": [
+    {
+      "name": "Nomic Foundation",
+      "email": "packages@nomic.foundation"
+    },
+    {
+      "name": "Antony Blakey",
+      "email": "antony@nomic.foundation"
+    },
+    {
+      "name": "Igor Matuszewski",
+      "email": "igor@nomic.foundation"
+    },
+    {
+      "name": "Omar Tawfik",
+      "email": "omar@nomic.foundation"
+    }
+  ],
+  "type": "commonjs",
+  "devDependencies": {
+    "@napi-rs/cli": "2.16.1"
+  },
+  "optionalDependencies": {
+    "@slang-private/slang-testlang-darwin-arm64": "0.13.1",
+    "@slang-private/slang-testlang-win32-arm64-msvc": "0.13.1",
+    "@slang-private/slang-testlang-linux-arm64-gnu": "0.13.1",
+    "@slang-private/slang-testlang-linux-arm64-musl": "0.13.1",
+    "@slang-private/slang-testlang-win32-ia32-msvc": "0.13.1",
+    "@slang-private/slang-testlang-darwin-x64": "0.13.1",
+    "@slang-private/slang-testlang-win32-x64-msvc": "0.13.1",
+    "@slang-private/slang-testlang-linux-x64-gnu": "0.13.1",
+    "@slang-private/slang-testlang-linux-x64-musl": "0.13.1"
+  },
+  "napi": {
+    "triples": {
+      "defaults": false,
+      "additional": [
+        "aarch64-apple-darwin",
+        "aarch64-pc-windows-msvc",
+        "aarch64-unknown-linux-gnu",
+        "aarch64-unknown-linux-musl",
+        "i686-pc-windows-msvc",
+        "x86_64-apple-darwin",
+        "x86_64-pc-windows-msvc",
+        "x86_64-unknown-linux-gnu",
+        "x86_64-unknown-linux-musl"
+      ]
+    }
+  },
+  "engines": {
+    "node": ">= 10"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
```

Comment:
To avoid publishing this package, I suggest removing `publishConfig` here, and adding `private: true` above.

```suggestion
```